src/phoenix/trace/span_json_decoder.py

```diff
@@ -10,6 +11,22 @@
     SpanKind,
     SpanStatusCode,
 )
+from phoenix.trace.semantic_conventions import (
+    INPUT_MIME_TYPE,
+    OUTPUT_MIME_TYPE,
+    MimeType,
+)
+
+
+def json_to_attributes(obj: Optional[Dict[str, Any]]) -> Dict[str, Any]:
+    if obj is None:
+        return {}
+    assert isinstance(obj, dict)
```

Comment:
Typical to use assert statements rather than raising errors?

Reply:
changed to raise